os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -66,6 +67,15 @@ static int ns_flash_write(uint32_t address, uint32_t len, uint8_t *data)
 	return ret;
 }
 
+static void ns_setstatusbits(u32 NewState)
+{
+	device_mutex_lock(RT_DEV_LOCK_FLASH);
+	FLASH_Write_Lock();
+	FLASH_SetStatusBits(0x2c, NewState);
```

Comment:
Is there any use case that calling `FLASH_SetStatusBits()` without `FLASH_Write_Lock()`?
If there is no, how about adding lock and unlock inside the `FLASH_SetStatusBits()`? Then we can reduce the code.
When I checked, all `FLASH_SetStatusBits` are binded with `FLASH_Write_Lock` and `FLASH_Write_Unlock`.
And many functions related to `FLASH_XXX` have lock and unlock inside.

Reply:
FLASH_SetStatusBits() is a API located in ROM, we cannot modify it.